student.rb

```diff
@@ -1,12 +1,13 @@
 require './person'
 
 class Student < Person
```

Comment:
- [ ] OPTIONAL
Kindly mark _classroom_ as an `attr_reader`, so that it can be accessed and updated